### DISCUSSION TITLE
Unable to change the color of the pin input to white in dark mode

### DISCUSSION THREAD

**Discussion by devhims** [chakra-ui/chakra-ui#3380] [Q&A]
I'm trying to change the outline color of the pin input fields to white in dark mode. However, I couldn't find a prop for that. Could it be done? 

<img width="470" alt="Screenshot 2021-02-14 at 2 17 02 PM" src="https://user-images.githubusercontent.com/17825870/107872340-b510e100-6ecf-11eb-8772-61ca21c5953c.png">

**Comment by devhims** [comment]
I managed to get a somewhat better outcome with the `borderColor` prop on the `PinInputField` children. But the inner circle remains unaffected. 

<img width="475" alt="Screenshot 2021-02-14 at 2 59 26 PM" src="https://user-images.githubusercontent.com/17825870/107873054-61a19180-6ed5-11eb-9aa3-64f262d2da7d.png">

**Comment by with-heart** [comment]
Hi @devhims! The ○ character in each field is a `placeholder`, so you could change the color in a few ways:

- If you only want this style in one place, add the `_placeholder` style to each `PinInputField`: https://codesandbox.io/s/eloquent-mestorf-eb4xv?file=/src/App.tsx
- If you want this style on any `PinInput` in your app, add the `_placeholder` style to `theme.components.PinInput.variants.outline` (the default `variant` for `PinInput`): https://codesandbox.io/s/objective-shadow-2f20k?file=/src/App.tsx

Also just want to throw out that at the top of each component page, there is a "View Theme Source" button that allows you to see the actual default theme styles for each component. That's usually fairly helpful for figuring out how to style things, or to see what you might need to override in the default theme in order to override a style.

![image](https://user-images.githubusercontent.com/1954752/107883988-4ed09200-6ec0-11eb-9f27-453edc0d9407.png)

Hope that helps!

**Reply by devhims** [reply]
Thanks @with-heart! Both suggestions worked, and I decided to go with the second option.  :)

Since we're on the topic of customizing the theme, can you also share a reference on how we can change the `colorScheme` for `PinInput`. I checked the theme source, but couldn't figure it out.

I'd like to try some of the possible options mentioned on the page.

<img width="711" alt="Screenshot 2021-02-14 at 11 52 32 PM" src="https://user-images.githubusercontent.com/17825870/107885358-cc77ba80-6f1f-11eb-8e21-6fea5c277eff.png">
